catroid/src/main/java/org/catrobat/catroid/content/bricks/DroneMoveBackwardBrick.java

```diff
@@ -40,81 +37,45 @@
 	private static final long serialVersionUID = 1L;
 
 	public DroneMoveBackwardBrick(int durationInMilliseconds, int powerInPercent) {
-		initializeBrickFields(new Formula(durationInMilliseconds / 1000.0), new Formula(powerInPercent));
+		this(new Formula(durationInMilliseconds / 1000.0), new Formula(powerInPercent));
 	}
```

Comment:
All bricks need to call `addAllowedBrickField` to set up the mapping correctly during deserialising now. So all Formula Bricks need to have an empty constructor which calls this(…).

I guess adding an empty constructor is the easiest solution for now. (readResolve won't do the trick because it sets an empty formula into the formulaMap)

Reply:
oh, yes. that's an interesting one -> have been thinking about this yesterday. and I agree, for now this seems to be the best solution.